### ISSUE TITLE
Pass Saxon instance to Morgana

### ISSUE THREAD

**Issue by JDziurlaj** [xml-project/support-for-xmleditor#3]
Right now you must put a copy of Saxon in the Morgana directory. However, you should be able to just pass thru the copy of Saxon used in Oxygen directly,as the Calabash plugin does.

**Comment by xml-project** [comment]
Right. I will implement this as an option for MorganaXProc-III. This adapter is for 1.0 which I regard as legacy. So I will not fix this.

**Comment by JDziurlaj** [comment]
Are there any plans to update this plugin for MorganaXProc-III?
